pkg/kubectl/cmd/create_role_test.go

```diff
@@ -77,6 +77,23 @@ func TestCreateRole(t *testing.T) {
 				},
 			},
 		},
+		"test-subresources": {
+			verbs:     "get,watch,list",
+			resources: "pods/status",
```

Comment:
How would you express a subresource of a resource in an API group? The parsing in Complete() wouldn't handle `rs.extensions/scale` correctly